src/compas/geometry/surfaces/nurbs.py

```diff
@@ -320,20 +320,32 @@ def from_step(cls, filepath):
         return new_nurbssurface_from_step(cls, filepath)
 
     @classmethod
-    def from_fill(cls, curve1, curve2):
-        """Construct a NURBS surface from the infill between two NURBS curves.
+    def from_fill(cls, curve1, curve2, curve3=None, curve4=None, style='stretch'):
+        """Construct a NURBS surface from the infill between two, three or four contiguous NURBS curves.
 
         Parameters
         ----------
-        curve1 : :class:`~compas.geometry.NurbsCurve`
-        curve2 : :class:`~compas.geometry.NurbsCurve`
+        curve1 : :class:`~compas_occ.geometry.OCCNurbsCurve`
+        curve2 : :class:`~compas_occ.geometry.OCCNurbsCurve`
+        curve3 : :class:`~compas_occ.geometry.OCCNurbsCurve`, optional.
+        curve4 : :class:`~compas_occ.geometry.OCCNurbsCurve`, optional.
+        style : Literal['stretch', 'coons', 'curved'], optional.
+            The fill style:
+            'stretch' gives the flattest patch.
+            'coons' gives a rounded patch with less depth than that of curved.
+            'curved' gives with the most rounded patch.
```

Comment:
this will render awkwardly.
perhaps
```
style : Literal[...], optional

    * ``'stretch'`` produces the flattest patch.
    * ``'curved'`` produces a rounded patch.
    * ``'coons'`` is between stretch and coons.

```
the blank lines are important in this case